python/cudf/cudf/tests/test_dataframe_copy.py

```diff
@@ -160,7 +139,7 @@ def test_kernel_deep_copy():
     cdf = gdf.copy(deep=True)
     sr = gdf["b"]
 
-    add_one[1, len(sr)](sr._column.data_array_view(mode="write"))
+    cp.asarray(sr._column)[1] = 42
     assert not gdf.to_string().split() == cdf.to_string().split()
```

Comment:
Use `!=` instead of `not ... ==`. Also does this need to be converted to a string? That feels unnecessary... here are some options:
```suggestion
    assert gdf.to_string().split() != cdf.to_string().split()
```
```suggestion
    assert gdf != cdf
```
Or to match the test below, use `assert_neq`.
```suggestion
    assert_neq(gdf, cdf)
```